src/components/Tool/GoogleImport/GoogleImport.tsx

```diff
@@ -0,0 +1,553 @@
+                                  fullWidth
+                                  loading={contactTagsListLoading}
+                                  filterSelectedOptions
+                                  value={tagsForAllList}
+                                  options={
+                                    contactTagsList?.accountList
+                                      ?.contactTagList || []
+                                  }
+                                  renderInput={(params): ReactElement => (
+                                    <ContactTagInput
+                                      {...params}
+                                      placeholder={t('add tag')}
+                                      disabled={isSubmitting}
+                                    />
+                                  )}
+                                  onChange={(_, tagsForAllList): void =>
+                                    setFieldValue(
+                                      'tagsForAllList',
+                                      tagsForAllList,
+                                    )
+                                  }
+                                />
+                                <FormControl>
+                                  <RadioGroup
+                                    aria-labelledby="demo-radio-buttons-group-label"
+                                    defaultValue="false"
+                                    name="override"
+                                    onChange={handleChange('override')}
+                                    value={override}
+                                  >
+                                    <FormControlLabel
+                                      value="false"
+                                      control={<Radio />}
+                                      label={t(
+                                        'This import should only fill blank fields in current contacts and/or add new contacts.',
+                                      )}
+                                    />
+                                    <FormControlLabel
+                                      value="true"
+                                      control={<Radio />}
+                                      label={t(
+                                        'This import should override all fields in current contacts (contact info, notes) and add new contacts.',
+                                      )}
+                                    />
+                                  </RadioGroup>
+                                </FormControl>
+                              </Section>
+                            </Box>
+                            <CardActions>
+                              <Button
+                                variant="contained"
+                                disabled={!isValid || isSubmitting}
+                                onClick={() => {
+                                  importAll
```

Comment:
Let's use if/else instead of a ternary since we're not doing anything with the return values.

Reply:
Ok. Why would that be bettery than ternary here?

Reply:
I don't mind, but I see Caleb C's point. When using an if/else statement to decide which function to run, it does make it easier to read if it is in a if/else statement.